modern/src/reports/ReportLayoutPage.js

```diff
@@ -18,7 +18,7 @@ const useStyles = makeStyles(theme => ({
   },
 }));
 
-const ReportLayoutPage = ({ reportFilterForm:ReportFilterForm, setItems, children }) => {
+const ReportLayoutPage = ({ reportFilterForm:ReportFilterForm, setItems, type, setType, children }) => {
```

Comment:
I don't like that we need to pass through a parameter that is not generic. We need to fix that somehow. Maybe component breakdown should be different.